Tests/LibWeb/Layout/input/grid/intrinsic-sized-column.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <style>
```

Comment:
You need to specify `SerenitySans` to have consistent font metrics on all systems.

Reply:
Oo right! Thanks :)

I guess it'd be possible to apply this font on all layout tests via the runner right? This way wouldn't have to manually indicate this font every time?

Reply:
yeah, it would be nice. I think we can start from throwing an error in the runner if font is not `SerenitySans` and that would be a good start to make it less confusing for people unfamiliar with this convention :)

Reply:
I'm thinking of creating a temp file where I add the font style declaration to it and use that as the input html file.

Which I guess would make it not necessary to emit an error.